core/modules/editor/engines/framed.js

```diff
@@ -172,6 +193,20 @@ FramedEngine.prototype.handleFocusEvent = function(event) {
 	if(this.widget.editCancelPopups) {
 		$tw.popup.cancel(0);	
 	}
+	var currentTiddler = this.widget.document.querySelector('[data-tiddler-title="' + CSS.escape(this.widget.editTitle) + '"].tc-tiddler-frame');
```

Comment:
`data-tiddler-title` doesn't need to be excaped. That's the advantage of `data-tiddler`

Reply:
Imo the escaping is needed

Reply:
The ViewTemplate doesn't do any escaping. So the code won't work https://tiddlywiki.com/#%24%3A%2Fcore%2Fui%2FViewTemplate

Why do you think css-escaping is needed for data-xxx ?

Reply:
the html-spec doesn't limit the "value" in any case: https://html.spec.whatwg.org/multipage/dom.html#attr-data-*

Reply:
I concur that the escaping is needed; otherwise we wouldn't be able to locate tiddlers with a single quote in their title.